clang/include/clang/Driver/Options.td

```diff
@@ -3750,6 +3750,11 @@ def fsycl_preserve_device_nonsemantic_metadata : Flag<["-"], "fsycl-preserve-dev
 def ftarget_compile_fast : Flag<["-"], "ftarget-compile-fast">,
   Visibility<[ClangOption, CLOption, DXCOption]>, HelpText<"Experimental feature: Reduce target "
   "compilation time, with potential runtime performance trade-off.">;
+def ftarget_device_link : Flag<["-"], "ftarget-device-link">,
```

Comment:
Is this option already approved by the WG? I was just thinking -fsycl-early-device-link sounded a bit better.

Thanks

Reply:
The option was just approved...